connectors/katalog/pkg/connector/connector_test.go

```diff
@@ -119,86 +118,104 @@ func TestCreateAsset(t *testing.T) {
 			},
 		},
 	}
-	var csvFormat taxonomy.DataFormat = "csv"
-	sourceAssetName := "paysim-csv"
-	destAssetName := "new-paysim-csv"
-	destCatalogID := "fybrik-system"
-
-	// Create a fake request to Katalog connector
-	createAssetReq := &datacatalog.CreateAssetRequest{
-		DestinationCatalogID: destCatalogID,
-		DestinationAssetID:   destAssetName,
-		ResourceMetadata: datacatalog.ResourceMetadata{
-			Name: sourceAssetName,
-			Columns: []datacatalog.ResourceColumn{
-				{
-					Name: "nameDest",
-					Tags: &taxonomy.Tags{Properties: serde.Properties{Items: map[string]interface{}{
-						"PII": true,
-					}}},
-				},
-				{
-					Name: "nameOrig",
-					Tags: &taxonomy.Tags{Properties: serde.Properties{Items: map[string]interface{}{
-						"SPI": true,
-					}}},
-				},
-			},
+	var tests = []struct {
+		csvFormat       taxonomy.DataFormat
+		sourceAssetName string
+		destAssetName   string
+		destCatalogID   string
+	}{
+		{
+			csvFormat:       "csv",
+			sourceAssetName: "paysim-csv",
+			destAssetName:   "new-paysim-csv",
+			destCatalogID:   "fybrik-system",
 		},
-		Details: datacatalog.ResourceDetails{
-			Connection: s3Connection,
-			DataFormat: csvFormat,
+		{
+			csvFormat:       "csv",
+			sourceAssetName: "a%ba/df12-csv",
+			destAssetName:   "a%ba/d//f=12-csv",
```

Comment:
This one should fail. You can't create Asset with name "d//f=12-csv"

Reply:
The test doesn't fail b/c internally it makes it a valid name.